packages/ui/src/Button/index.tsx

```diff
@@ -200,6 +201,7 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
             ? theme.borderRadius.sm
             : theme.borderRadius.full
         }
+        {...props}
```

Comment:
In Radix, `asChild` prop will only work if the nested component passes the rest props. 

It doesn't mean we allow users to pass whatever they want - at least TypeScript disagrees with it

Reply:
Ah! Great solution. I was kinda bothered by the fact that I wasn't previously spreading remaining props when Radix wants us to, but I didn't want to change it for TypeScript reasons.

request: could you add a note documenting this? so that future maintainers don't think that there should be add'l props in TypeScript?